src/Makefile

```diff
@@ -36,19 +41,22 @@ OPTFILES = opt-print-ustrade.tex \
 			opt-scala.tex
 
 # All the LaTeX files for the CTFP book in order of dependency
-TEXFILES = $(TOPTEXFILES) $(SCALATEXFILES) $(OCAMLTEXFILES) $(OPTFILES)
+TEXFILES = $(TOPTEXFILES) $(SCALATEXFILES) $(OCAMLTEXFILES) $(REASONTEXFILES) $(OPTFILES)
 
 default: suffix=''
 default: out-dir $(DEFAULTPDF) # todo cover
 
-all: default scala ocaml
+all: default scala ocaml reason
 
 scala: suffix='-scala'
 scala: out-dir version.tex $(SCALAPDF)
 
 ocaml: suffix='-ocaml'
 ocaml: out-dir version.tex $(OCAMLPDF)
 
+reason: suffix='-reason'
+reason: out-dir version.tex $(REASONPDF)
```

Comment:
due to me being dumb with makefiles, please add a `clean` command here before `out-dir` (see https://github.com/hmemcpy/milewski-ctfp-pdf/commit/78971a95b52e8dc3790194c22b2d7a2e15790955) (also in `all`)